STM32F405RGTx/Core/Src/timer_utils.c

```diff
@@ -0,0 +1,32 @@
+#include <stdint.h>
+#include "main.h"
+#include "timer_utils.h"
+
+void set_led_intensity(uint8_t color, float intensity)
+{
+    uint32_t ccr_val = (uint32_t)(((100 - intensity) * ARR_VAL) / 100);
+    switch (color)
+    {
+    case 1:
+        htim1.Instance->CCR1 = ccr_val;
```

Comment:
These should be changed to `htim3`

Reply:
also, these switch statements need `break;`'s

Reply:
@kani-tech You still have not addressed this